data/low_income_thresholds.json

```diff
@@ -286,6 +286,71 @@
         "7": 108750,
         "8": 115800
       }
+    },
+    "nv-csa-reno": {
+      "incentives": [
+        "NV-74"
+      ],
+      "source_url": "https://www.csareno.org/Energy-Efficiency-Weatherization",
+      "thresholds": {
+        "1": 27180,
+        "2": 36620,
+        "3": 46060,
+        "4": 55500,
+        "5": 64940,
+        "6": 74380,
+        "7": 83820,
+        "8": 93260
+      }
+    },
+    "nv-rural-nevada-development-corporation": {
+      "incentives": [
+        "NV-75"
+      ],
+      "source_url": "https://aspe.hhs.gov/topics/poverty-economic-mobility/poverty-guidelines",
+      "thresholds": {
+        "1": 22590,
+        "2": 30660,
+        "3": 38730,
+        "4": 46800,
+        "5": 54870,
+        "6": 62940,
+        "7": 71010,
+        "8": 79080
+      }
+    },
+    "nv-nevada-rural-housing": {
+      "incentives": [
+        "NV-76",
+        "NV-79"
+      ],
+      "source_url": "https://nvrural.org/programs/weatherization/",
+      "thresholds": {
+        "1": 27180,
+        "2": 36620,
+        "3": 46060,
+        "4": 55500,
+        "5": 64940,
+        "6": 74380,
+        "7": 83820,
+        "8": 93260
+      }
+    },
+    "nv-help-of-southern-nevada": {
+      "incentives": [
+        "NV-77"
+      ],
+      "source_url": "https://static1.squarespace.com/static/5e91c64d83735c5db90418a6/t/65dd07f61363a972f9bc5e65/1708984310854/WEBSITES+INCOME+SHEET+-+2024+WX.pdf",
+      "thresholds": {
+        "1": 30120,
+        "2": 40880.04,
+        "3": 51639.96,
+        "4": 62400,
+        "5": 73160.04,
+        "6": 83919.96,
+        "7": 94680,
+        "8": 105440
```

Comment:
I'm pretty sure this is just rounding error -- e.g. the source PDF lists the monthly limit for 3 people as 4303.33. That probably comes from dividing 51640 / 12, which is 4303.33.... ([repeating of course](https://youtu.be/mLyOj_QD4a4?t=68)). So I think we should just round these to the nearest whole dollar. I've never seen fractional income limits anywhere else.